_notebooks/2021-05-04-spectral-svd.ipynb

```diff
@@ -0,0 +1,987 @@
+{
```

Comment:
<p>Instead of frobenius norm, say "error"</p><p> </p><p>title can reflect the problem you are looking at, no need to repeat the names of the x and y axes</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neurodata/notebooks/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>